presto-main/src/main/java/io/prestosql/sql/planner/planprinter/IoPlanPrinter.java

```diff
@@ -514,6 +671,41 @@ else if (writerTarget instanceof CreateReference || writerTarget instanceof Inse
             return processChildren(node, context);
         }
 
+        private EstimatedStatsAndCost getEstimatedStatsAndCost(TableScanNode node, TupleDomain<ColumnHandle> predicate, TableMetadata tableMetadata)
+        {
+            QualifiedObjectName tableName = new QualifiedObjectName(
+                    tableMetadata.getCatalogName().getCatalogName(),
+                    tableMetadata.getTable().getSchemaName(),
+                    tableMetadata.getTable().getTableName());
+            Optional<TableHandle> optTableHandle = metadata.getTableHandle(session, tableName);
+            double bytesScanned = 0;
```

Comment:
* can you please extract the below if block as method (getBytesScannedStatistics). 
* Also in case where there is no table handle or no stats, please use `Double.NaN`. Using `0` could be misleading.